end-to-end-test/mocks/general/index.ts

```diff
@@ -0,0 +1,13 @@
+import mock_exchange_rates from './exchange_rates';
+import mock_payout_currencies from './payout_currencies';
+import mock_time from './time';
+import mock_website_status from './website_status';
+
+const general = context => {
```

Comment:
```suggestion
const general = (context: Context) => {
```

Can we define type for this? :thinking: